Dockerfile_dev

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.13 as certs
+RUN apk --update add ca-certificates
+
+FROM golang:1.16.4
+ARG BUILD_TAG=latest
+ENV TAG $BUILD_TAG
+ARG USER_UID=10001
+USER ${USER_UID}
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY otelcol-sumo /otelcol-sumo
```

Comment:
Sorry if this is a stupid question but where did this `otelcol-sumo` file come from?

Reply:
Oh I think I get it, it's being built in CI (GH Actions) in the `build` job and then copied in the `build-container-images` job.

Reply:
Yes. The reason for this is that I tried to do it the same way as for fluentd (run all builds and wait for build pids) but the CI would stall for an hour without completing: https://github.com/SumoLogic/sumologic-otel-collector/actions/runs/928636321

Then I tried to split this and use the github provided builders for arm (using strategy matrix for that) but then amd 64 build would build in handful of minues (👍 ) but arm builds would stall and not complete even within 30-40m (sometimes they would complete but we don't want to wait 30m for a build 😬 ) https://github.com/SumoLogic/sumologic-otel-collector/actions/runs/929008398

Reply:
Ohhh this sucks. Thanks for explaining.